dev/Cargo.toml

```diff
@@ -8,7 +8,7 @@ publish = false
 anyhow = "1.0.62"
 async-nats = { git = "https://github.com/nats-io/nats.rs.git", rev = "261ead4" }
 bollard = "0.14.0"
-chrono = { version="0.4.22", default_features = false }
+chrono = { version="0.4.22", default-features = false }
```

Comment:
good catch :)

Reply:
Not me, cargo told me when I removed lazy_static. (Now it didn't tell me when I added lazy_static, so not sure ehats up there)